.github/workflows/ui.yml

```diff
@@ -0,0 +1,13 @@
+name: Centralized workflow
+on:
+  - push
+  - pull_request
+  - workflow_dispatch
+
+jobs:
+  ui:
+    uses: folio-org/.github/.github/workflows/ui.yml@v1
```

Comment:
Just double checking, to be sure... Is `.github/.github` right or should there just be 1 level of `.github`?

Reply:
It's right...the workflows have to go in the `.github` folder, but we named the repository `.github`